docs/web/howto/client.rst

```diff
@@ -331,74 +331,85 @@ This function returns a ``Deferred`` that fires with the body after the request
 
 
 
-HTTP over SSL
-~~~~~~~~~~~~~
-
-
-    
-
-Everything you've read so far applies whether the scheme of the request
-URI is *HTTP* or *HTTPS* .  However, to control the SSL
-negotiation performed when an *HTTPS* URI is requested, there's
-one extra object to pay attention to: the SSL context factory.
-
-
-
-
-
+Customizing your HTTPS Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-``Agent`` 's constructor takes an optional second argument, a context factory.
-Generally you don't want to create the context factory yourself: it's easy to get wrong.
-Instead, Twisted provides helpers for building correctly configured ones.
-For example, you can use :api:`twisted.internet.ssl.optionsForClientTLS <optionsForClientTLS>`, which takes care of all hostname certificate verification for you.
+Everything you've read so far applies whether the scheme of the request URI is *HTTP* or *HTTPS* .
 
+By default, since version 15.0.0 (if you're using an earlier version, please upgrade, as this is a critical security feature!), Twisted will automatically validate ``https`` URLs against your platform's trust store to the best of its ability.
```

Comment:
"By default" and "automatically" feel somewhat redundant.  I'd say "Twisted will automatically validate ..." or "By default, Twisted will validate ..."